src/routes/(site)/teams/team-section.svelte

```diff
@@ -275,7 +275,7 @@
     animation-duration: 1.8s;
     animation-name: slide;
     animation-iteration-count: infinite;
-    animation-direction: alternate;
+    animation-direction: alternate; 
```

Comment:
Nit: Random space.
```suggestion
    animation-direction: alternate;
```

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1058/commits/813db728c4690a466a8b291db83c0d6fdb8f1bb8> by @karnikaavelumani.